doc/source/train/examples/lightning/lightning_cola_advanced.ipynb

```diff
@@ -294,41 +300,63 @@
    "outputs": [],
    "source": [
     "if SMOKE_TEST:\n",
-    "    lightning_config = (\n",
-    "        LightningConfigBuilder()\n",
-    "        .module(cls=SentimentModel, lr=1e-5, eps=1e-8)\n",
-    "        .trainer(max_epochs=2, accelerator=\"gpu\")\n",
-    "        .checkpointing(save_on_train_epoch_end=False)\n",
-    "        .build()\n",
-    "    )"
+    "    train_func_config[\"max_epochs\"] = 2\n",
+    "    for split, ds in ray_datasets.items():\n",
+    "        ray_datasets[split] = ds.random_sample(0.1)"
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Fine-tune the model with LightningTrainer\n",
+    "# Distributed training with Ray TorchTrainer\n",
     "\n",
-    "Train the model with the configuration we specified above. \n",
+    "Next, define a {class}`TorchTrainer <ray.train.torch.TorchTrainer>` to launch your training function on 4 GPU workers. \n",
     "\n",
-    "To feed data into LightningTrainer, we need to configure the following arguments:\n",
+    "Here, you can pass the full Ray dataset to `TorchTrainer(datasets=)`. TorchTrainer automatically shards the dataset among multiple workers according to `DataConfig`.\n",
```

Comment:
1. Get rid of DataConfig in code since it's already split
2. nit: I don't know if it should be `TorchTrainer(datasets)` or just "the `datasets` argument", but I feel like the current way is a little off.